src/rules/declaration-block-no-duplicate-properties/__tests__/index.js

```diff
@@ -60,3 +60,33 @@ testRule(rule, {
     message: messages.rejected("color"),
   } ],
 })
+
+testRule(rule, {
+  ruleName,
+  config: [ true, { ignore: ["consecutive-duplicates"] } ],
+  skipBasicChecks: true,
+
+  accept: [ {
+    code: "p {font-size: 16px; font-size: 1rem; }",
```

Comment:
Minor nit-pick, we tend to put a space after the opening brace (unless we're specifically testing for some variant of that). The consistency makes it easier to scan the tests across files.